db/migrate/20170101185242_add_venue_id_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddVenueIdToEvents < ActiveRecord::Migration[5.0]
+  def change
+    add_column :events, :venue_id, :integer
```

Comment:
is it possible to have an event without venue? I don't think so.
if you make it mandatory, you won't need all the nil checks - profit. also won't need that helper method in this case.